packages/date/package.json

```diff
@@ -32,15 +32,15 @@
   },
   "peerDependencies": {
     "@availity/form": "^0.1.0",
-    "@availity/yup": "^2.0.0",
+    "@availity/yup": "^3.0.2",
     "formik": "^2.0.1",
     "react": "^16.3.0",
     "reactstrap": "^8.0.0 || ^7.0.0 || ^6.0.0",
     "yup": "^0.27.0"
   },
   "dependencies": {
     "@availity/icon": "^0.7.5",
-    "@availity/yup": "^2.0.0",
+    "@availity/yup": "^3.0.2",
```

Comment:
I think this should be a `devDep`. As its already in peerDeps since we don' require it. Otherwise lets remove it from peerDeps completely.

Also this would be a breaking change since we are forcing the user to install a major version bump.